vesc_driver/package.xml

```diff
@@ -21,7 +21,9 @@
   <depend>rclcpp</depend>
   <depend>rclcpp_components</depend>
   <depend>std_msgs</depend>
+  <depend>geometry_msgs</depend>
   <depend>vesc_msgs</depend>
+  <depend>sensor_msgs</depend>
```

Comment:
It looks like you intended to publish `sensor_msgs/msg/Imu`. Can you please add this functionality? This is significantly more helpful than a VESC-specific message.

Reply:
Sorry for the long delay. `sensor_msgs/msg/Imu` is a standard message and lots of stacks will accept this message. Please either change to publishing this message or at least add it as a second output.